client/src/components/AdminCard/index.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { Button, Pagination } from 'antd';
+import './style.css';
+
+const AdminCard = ({
+  imgUrl,
+  name,
+  description,
+  githbUrl,
+  student,
+  websiteLink,
+  studentNames,
+  ComProject,
+  CliProject,
+  editCard,
+  deleteCard,
+}) => {
+  return (
+    <>
+      <section className="card-container">
+        <ul>
+          <li>
+            <div className="name-img-card">
+              <div className="img-card">
+                <img src={imgUrl} alt="card" />
+              </div>
+              <div>
+                <h3>Name</h3>
+                <p>{name}</p>
+              </div>
+            </div>
+            {description && (
+              <div>
+                <h3>Description</h3>
+                <p>{description}</p>
+              </div>
+            )}
+            <div>
+              <h3>Github link</h3>
+              <NavLink exact to={githbUrl}>
+                Click
+              </NavLink>
+            </div>
+            {student && (
+              <div>
+                <h3>Student</h3>
+                <NavLink exact to="/admin/students">
+                  View
+                </NavLink>
+              </div>
+            )}
+            {websiteLink && (
+              <div>
+                <h3>Website</h3>
+                <NavLink exact to={websiteLink}>
+                  View
+                </NavLink>
+              </div>
+            )}
+            {studentNames && (
+              <div>
+                <h3>Student</h3>
+                <p>{studentNames}</p>
+              </div>
+            )}
+            {ComProject && (
+              <div>
+                <h3>Community p</h3>
+                <NavLink exact to="/admin/projects?type=internal">
```

Comment:
Should this link to a page which shows just this cohort's projects, instead of all projects for all cohorts?  If so, you might need to put a cohortId into the link url somewhere.
I think the same question applies to the students link (line 49) and the client projects link (line 79).

Reply:
Oh yeah, I forget that. Thank you 😊. I will correct this big error now.
```js
<Link to="/admin/projects/:cohortId?type=internal">View</Link>
```

Reply:
```js
<Link to={`/admin/projects/:${internalProject}?type=internal`}>
```